repository-data/application/src/main/resources/hcm-config/configuration/queries/templates/new-bsh-comparator-feature.yaml

```
@@ -0,0 +1,11 @@
+definitions:
+  config:
+    /hippo:configuration/hippo:queries/hippo:templates/new-bsh-comparator-feature:
+      jcr:primaryType: hippostd:templatequery
+      hippostd:icon: adddocument_ico
+      hippostd:modify: [./_name, $name, './hippotranslation:locale', $inherited, './hippotranslation:id',
+        $uuid, './hippostdpubwf:createdBy', $holder, './hippostdpubwf:creationDate',
```

YAMLlint (1.37.1)

[error] wrong indentation: expected 24 but found 8

(indentation)